### PR TITLE
Update scheduler docs to match latest Helm chart.

### PR DESCRIPTION
**Pull request by artursouza**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md [hunk 1]
```diff
@@ -6,12 +6,15 @@ weight: 50000
 description: "Configure Scheduler to persist its database to make it resilient to restarts"
 ---
 
-The [Scheduler]({{< ref scheduler.md >}}) service is responsible for writing jobs to its embedded database and scheduling them for execution.
-By default, the Scheduler service database writes this data to an in-memory ephemeral tempfs volume, meaning that **this data is not persisted across restarts**. Job data will be lost during these events.
+The [Scheduler]({{< ref scheduler.md >}}) service is responsible for writing jobs to its embedded Etcd database and scheduling them for execution.
+By default, the Scheduler service database writes this data to a Persistent Volume Claim of 1Gi of size using the cluster's default [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/). It means that there is no additional parameter required to run the scheduler service reliably on most Kubernetes deployments, although you will need additional configuration in some deployments or for a production environment.
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md:10]:
```suggestion
By default, the Scheduler service database writes this data to a Persistent Volume Claim of 1Gb of size using the cluster's default [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/). This means that there is no additional parameter required to run the scheduler service reliably on most Kubernetes deployments, although you will need additional configuration in some deployments or for a production environment.
```

**Review by msfussell — COMMENTED** [review]
Question - Does this PR replace https://github.com/dapr/docs/pull/4303/files

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md (hunk 1 above)
Comment (artursouza) [reply daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md:10]: @elena-kolevska is taking these changes to her PR.

Also, it is 1Gi or (1GiB), not 1Gb.

Comment (msfussell) [reply daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md:10]: Gi is the same a Gb. And is this PR now redundant due to this PR https://github.com/dapr/docs/pull/4306 anyway?

**Pull request closed without merging**